Loop Status Extension/StatusViewController.swift

```diff
@@ -20,33 +20,65 @@ class StatusViewController: UIViewController, NCWidgetProviding {
     @IBOutlet weak var batteryHUD: BatteryLevelHUDView!
     @IBOutlet weak var subtitleLabel: UILabel!
     
+    var context: StatusExtensionContext?
+    var defaults: UserDefaults?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         subtitleLabel.alpha = 0
         subtitleLabel.textColor = UIColor.secondaryLabelColor
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        defaults = UserDefaults(suiteName: Bundle.main.appGroupSuiteName)
+        if let defaults = defaults {
+            defaults.addObserver(
+                self,
+                forKeyPath: defaults.statusExtensionContextObservableKey,
+                options: [.new, .initial],
```

Comment:
If you aren't using the change dict in the observer than you should leave these blank.